### PR TITLE
feat: Add Nova micro/lite/pro using Converse

### PR DESCRIPTION
**Pull request by roman-romanov-o**
Amazon just added their own [SOTA models](https://docs.aws.amazon.com/nova/latest/userguide/what-is-nova.html), and three of them are available in Converse API, that we support

### CODE REVIEW

File: tests/integration_tests/test_chat_completion.py [hunk 1]
```diff
@@ -287,16 +310,30 @@ def dial_recall_expected(r: ChatCompletionResult):
...
+        test_case(
+            name="dialog recall",
+            messages=[
+                user("my name is Anton"),
+                ai("nice to meet you"),
+                user("what's my name?"),
+            ],
+            max_tokens=32,
+            expected=dial_recall_expected,
+        )
```
Comment (adubovik) [inline tests/integration_tests/test_chat_completion.py]: Remove this if the Paris version works for other models.

File: aidial_adapter_bedrock/deployments.py [hunk 2]
```diff
@@ -3,6 +3,9 @@
 
 
 class ChatCompletionDeployment(str, Enum):
+    AMAZON_NOVA_PRO = "amazon.nova-pro-v1:0"
+    AMAZON_NOVA_LITE = "amazon.nova-lite-v1:0"
+    AMAZON_NOVA_MICRO = "amazon.nova-micro-v1:0"
```
Comment (adubovik) [inline aidial_adapter_bedrock/deployments.py:8]: 1. Update README
2. Update test_endpoints.py

File: tests/integration_tests/test_chat_completion.py (hunk 1 above)
Comment (roman-romanov-o) [reply tests/integration_tests/test_chat_completion.py]: Yep, turns out this works even for Amazon Titan and Cohere (which was excluded from `what's your name` test case)

Replaced this with Paris test case

File: aidial_adapter_bedrock/deployments.py (hunk 2 above)
Comment (roman-romanov-o) [reply aidial_adapter_bedrock/deployments.py:8]: Fixed

**Comment by roman-romanov-o** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-bedrock-pr-197.nightly-test.deltixhub.io
>E2E tests status: success

File: README.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ Note that a model supports `/truncate_prompt` endpoint if and only if it support
 |Stability AI|Stable Image Ultra 1.0|stability.stable-image-ultra-v1:0|text-to-image|❌|🟡|❌|
 |Stability AI|Stable Image Core 1.0|stability.stable-image-core-v1:0|text-to-image|❌|🟡|❌|
 |Amazon|Titan Text G1 - Express|amazon.titan-tg1-large|text-to-text|🟡|🟡|❌|
+|Amazon|Nova Pro|amazon.nova-pro-v1:0|text-to-text|🟡|🟡|✅|
+|Amazon|Nova Lite|amazon.nova-lite-v1:0|text-to-text|🟡|🟡|✅|
+|Amazon|Nova Micro|amazon.nova-micro-v1:0|text-to-text|🟡|🟡|❌|
```
Comment (adubovik) [inline README.md]: What about `image-to-text` modality?

Comment (roman-romanov-o) [reply README.md]: fixed

**Review by adubovik — DISMISSED** [review]

**Comment by roman-romanov-o** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-bedrock-pr-197.nightly-test.deltixhub.io
>E2E tests status: success

**Review by adubovik — APPROVED** [review]

**Pull request merged**